bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/v18/V18AssortedRpc.scala

```diff
@@ -0,0 +1,38 @@
+package org.bitcoins.rpc.client.v18
+import org.bitcoins.rpc.client.common.Client
+import org.bitcoins.rpc.jsonmodels.{
+  DeriveAddressesResult,
+  GetDescriptorInfoResult,
+  GetNodeAddressesResult,
+  ListWalletDirResult
+}
+import org.bitcoins.rpc.serializers.JsonSerializers._
+import play.api.libs.json._
+
+import scala.concurrent.Future
+
+trait V18AssortedRpc {
+  self: Client =>
+
+  def getNodeAddresses(
+      count: Option[Double]): Future[GetNodeAddressesResult] = {
+    bitcoindCall[GetNodeAddressesResult]("getNodeAddresses",
+                                         List(Json.toJson(count)))
+  }
+
+  def listWalletDir(): Future[ListWalletDirResult] = {
+    bitcoindCall[ListWalletDirResult]("listwalletdir")
+  }
+
+  def deriveAddresses(
```

Comment:
IMO it would make sense to make a trait for descriptor RPCs, and move `deriveAddresses` and `getDescriptorInfo` into that trait